rtw_stb_image.h

```diff
@@ -0,0 +1,88 @@
+#ifndef RTW_STB_IMAGE_H
+#define RTW_STB_IMAGE_H
+
+// Disable strict warnings for this header from the Microsoft Visual C++ compiler.
+#ifdef _MSC_VER
+    #pragma warning (push, 0)
+#endif
+
+#define STB_IMAGE_IMPLEMENTATION
+#define STBI_FAILURE_USERMSG
+#include "external/stb_image.h"
+
+#include <cstdlib>
+#include <iostream>
+
+class rtw_image {
+  public:
+    rtw_image() : data(nullptr) {}
+
+    rtw_image(const char* image_filename) {
+        // Loads image data from the specified file. If the RTW_IMAGES environment variable is
+        // defined, looks only in that directory for the image file. If the image was not found,
+        // searches for the specified image file first from the current directory, then in the
+        // images/ subdirectory, then the _parent's_ images/ subdirectory, and then _that_
+        // parent, on so on, for six levels up. If the image was not loaded successfully,
+        // width() and height() will return 0.
+
+        auto filename = std::string(image_filename);
+        auto imagedir = getenv("RTW_IMAGES");
+
+        // Hunt for the image file in some likely locations.
+        if (imagedir && load(std::string(imagedir) + "/" + image_filename)) return;
+        if (load(filename)) return;
+        if (load("images/" + filename)) return;
+        if (load("../images/" + filename)) return;
+        if (load("../../images/" + filename)) return;
+        if (load("../../../images/" + filename)) return;
+        if (load("../../../../images/" + filename)) return;
+        if (load("../../../../../images/" + filename)) return;
```

Comment:
Right.  I copied from textbook and totally forgot to delete these😭
Since all the images should be in the images file (can be moved or renamed), we only need "if (load("images/" + filename)) return;" line this case:))

Reply:
hard-coded part removed, this is the result
![image](https://github.com/cypraeno/caitlyn/assets/108316903/a9c5f373-fe98-4582-ac83-0b2959d28776)
User would enters smt like this v
![image](https://github.com/cypraeno/caitlyn/assets/108316903/a0d96859-1106-426b-88d8-83df1f2e50db)
I haven't pushed yet, wanted to confirm if this is what we want by letting the user specify a path and removeing the goofy part

Reply:
has been updated